influxdb_plugin.py

```diff
@@ -181,6 +183,35 @@ def _process_stat_list(stat_value, fields, tags, prefix=""):
                 _add_field(fields, item_name, list_value, value_type)
 
 
+def _influxdb_points_from_stat(stat_time, tags, stat_key, stat_value):
+    """
+    Create InfluxDB points/measurements from the stat query result.
+    """
+    points = []
+    point_tags = tags.copy()
+    fields = []
+    stat_value_type = type(stat_value)
+    if stat_value_type == list:
+        for stat in stat_value:
+            (fields, point_tags) = _influxdb_point_from_stat(
+                stat_time, tags, stat_key, stat)
+            points.append(
+                _build_influxdb_point(
+                    stat_time, point_tags, stat_key, fields))
+    elif stat_value_type == dict:
+        _process_stat_dict(stat_value, fields, point_tags)
+        points.append(
+            _build_influxdb_point(
+                stat_time, point_tags, stat_key, fields))
```

Comment:
Yea sorry, didn't notice that. I was just noticing that the first copy of point_tags created on line 191 is not used at all when stat_value_type == list. I think you could move the code from line 191 to inside the elif on line 201 and just past the if stat_value == "": clause on line 207.